<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TV_view</class>
 <widget class="QMainWindow" name="TV_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>780</width>
      <height>171</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
   </widget>
   <widget class="QLabel" name="bg00">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1026</width>
      <height>590</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="backgroundpic.qrc">:/new/prefix1/0.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="bg01">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1026</width>
      <height>590</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="backgroundpic.qrc">:/new/prefix1/1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="bg02">
    <property name="geometry">
     <rect>
      <x>1026</x>
      <y>0</y>
      <width>1026</width>
      <height>590</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="backgroundpic.qrc">:/new/prefix1/2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="bg03">
    <property name="geometry">
     <rect>
      <x>2052</x>
      <y>0</y>
      <width>1026</width>
      <height>590</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="backgroundpic.qrc">:/new/prefix1/3.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="openbutton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>400</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deletebutton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>460</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>储存</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adminbutton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>520</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QPushButton" name="freshButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>1</width>
      <height>1</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <zorder>bg00</zorder>
   <zorder>bg01</zorder>
   <zorder>bg02</zorder>
   <zorder>bg03</zorder>
   <zorder>openbutton</zorder>
   <zorder>deletebutton</zorder>
   <zorder>adminbutton</zorder>
   <zorder>tableView</zorder>
   <zorder>freshButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="backgroundpic.qrc"/>
 </resources>
 <connections/>
</ui>
